src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseParticipationResource.java

```diff
@@ -303,4 +336,8 @@ public ModelAndView redirectGetParticipationRepositoryFilesForCommitsDetailsView
         return new ModelAndView("forward:/api/repository/" + participation.getId() + "/files-content/" + commitId);
     }
 
+    public enum RepositoryType {
+        TEMPLATE, SOLUTION, TESTS
+    }
```

Comment:
This enum already exisits in de.tum.in.www1.artemis.domain.enumeration.RepositoryType.